### ISSUE TITLE
CLI: list all pending migrations

### ISSUE THREAD

**Issue by Saka7** [tgriesser/knex#3390]
# Feature request: Knex CLI list all pending migrations

Are there any plans for `list` command? The idea is to print all migrations which haven't been applied yet.

API Example: 
```sh
knex list

20190809115505_migration10
20190809115506_migrations11
20190809115507_migrations12
```
References:
- [sqitch](https://github.com/sqitchers/sqitch) - `sqitch status`
- [node-migrate](https://github.com/tj/node-migrate) - `migrate list`

**Comment by kibertoad** [comment]
@Saka7 Sounds useful! We would gladly accept PR for this.

**Comment by kibertoad** [comment]
Released in 0.19.3

**Comment by MohamedLamineAllal** [comment]
it's no more available ! Why was it removed !
